engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/views/register-deb-client.html

```diff
@@ -0,0 +1,32 @@
+<div ng-controller="ContentHostRegisterController"><li>
+  <p>Add the GPG public key used to sign deb packages from your system to download the Debian/Ubuntu client from your system:</p>
```

Comment:
Lets change `deb` to `DEB` This will be uniform then with this line and other areas where we have capitalized RPM:
`Install the pre-built bootstrap RPM:`

Reply:
Not sure about this one, since RPM is an acronym (which should be all capitalized), wheres `deb` package is conventionally written in all lower in reference to the `.deb` file extension used by most APT packages. Note that deb package is often used to refer to all APT based distributions like "Ubuntu packages" and not _just_ "Debian packages".

That being said, at the end of the day, I would be fine with either version ("deb" or "DEB").